setup.py

```diff
@@ -0,0 +1,64 @@
+
+"""Initial setup.py
+
+TODO: This file is very rudimentary and is setup purely to enable tox to
+run. None of the information in it reflects necessarily what it should be and
+will be updated to proper values by project owner.
+
```

Comment:
Yeah. So, I can see that there is a bit of a misunderstanding, no doubt because I didn't explain it properly. It isn't that tox uses a setup.py file for anything special. It uses it for the same thing that we always do, which is to build a package. Every single time tox runs, it will build a package and install it into its own virtual environments (it has one for each python version and task it performs). It does this to make sure that e.g. the test will be run exactly on the code that will be distributed (the package) and not the code which is in the directory. The distinction is subtle but really important.

In any case. Because tox actually uses it to build a package, then it is actually probably able to build a package. What **is**  missing are the trove classifiers (https://pypi.org/classifiers/), what **is probably** missing is read through by you of the meta data in ``__init__.py`` to make sure it is as you wish and what **may be** missing fairly soon is something on data-files, which you need to tell setup tools about separately.

I will add the comment at the top.

Reply:
Comment added.

Reply:
Thanks! The file looks great now.